### DISCUSSION TITLE
Radar filter not working

### DISCUSSION THREAD

**Discussion by andylaulh** [maca88/SmartBikeLights#42] [General]
Hi I'm trying to override the light pattern when the radar detects a car. However, the radar still uses its built-in flash pattern (too bright at night). This is my configuration:

`###0,610283536::1#7,5!Brake:1:2:0:0A[-25!Night:2:6:0:0Es0,r0I}0]0!Day:2:6:0:0Er0,s0I[70]0!On:1:6:0:0H]2!:1:0:0:0D=1##6,5:Gardia 300:0:16777215!2,:-1,Off:0!1,Low Solid:4!1,Group Ride:8!1,Night Flash:6!1,Day Flash:7#0::#0:0#123!:13!0,4,8#0#B3843##14#0#0`

Thank you very much for your help.

**Comment by maca88** [comment]
Hi,

based on [the manual](https://manuals.plus/bryton/gardia-r300-rear-light-manual#smart_control), Gardia r300 starts flashing when a vehicle is approaching you regardless the current mode. This is similar on how Varia works and unfortunately (at least for Varia), this is something that cannot be disabled.

**Reply by andylaulh** [reply]
Hi

Thank you very much for your prompt reply.

May I know how we can use the filter based on the radar detection then? I see that we can set filters based on the distance of vehicle detected and the threat level. Thanks!

**Reply by maca88** [reply]
Varia users are usually using the radar filter to turn on the light only when a vehicle is approaching in daytime in order to save battery life. In your case, you set the same light mode (Night Flash) when the vehicle is approaching (Day group) as when no vehicle is behind (On  group) for daytime. If you want to have the light always turned on in daytime, then you don't need the radar filter, unless you would like to switch to a brighter mode when a vehicle is approaching and then after the vehicle passes, switch back to the low intensity one.
I short, you correctly set the radar filter in Day and Night group, but as you set the same light mode for all of them, the light won't change the light mode, only the pattern will be changed when a vehicle is detected based on the manual (Night Flash -> Quickly Flashing Light).

**Reply by andylaulh** [reply]
> "but as you set the same light mode for all of them, the light won't change the light mode"

I have changed the light mode for the radar group so they are not the same: 

`###0,610283536::1#3,3!Radar:1:8:0:0I}1]0!On:1:6:0:0H]2!:1:0:0:0D=1##6,5:Gardia 300:0:16777215!2,:-1,Off:0!1,Low Solid:4!1,Group Ride:8!1,Night Flash:6!1,Day Flash:7#0::#0:0#123!:13!0,4,8#0#B3843##14#0#0`

Will test it out to see if the "Quick Flashing Light" pattern can be overridden. Thanks a lot.
